lazyflow/operators/ioOperators/opInputDataReader.py

```diff
@@ -289,10 +289,9 @@ def _attemptOpenAsUfmf(self, filePath):
     def _attemptOpenAsRESTfulPrecomputedChunkedVolume(self, filePath):
         if not filePath.lower().startswith("precomputed://"):
             return ([], None)
-        url = filePath.lstrip("precomputed://")
         reader = OpRESTfulPrecomputedChunkedVolumeReader(parent=self)
```

Comment:
probably not worth a change, but `OpRESTfulPrecomputedChunkedVolumeReader` seems to take `baseUrl` also as a parameter in `__init__`.

Reply:
It doesn't look like it takes it as a parameter, but worse - it compares `self.BaseUrl.value` (full input url) to `self._volume_object.volume_url` (prefix stripped). I guess the web adapter does need to keep a copy of the full url before stripping.

Reply:
This, and (more importantly) determining that it's a Precomputed URL in OpInputDataReader